### ISSUE TITLE
GeoPolygonQueryBuilderTests makes illegal minLon value

### ISSUE THREAD

**Issue by rmuir** [elastic/elasticsearch#16399]
When i upgrade lucene, things are pickier and the test fails. I'd rather @nknize or @mikemccand look into this as I know they have battle scars already.

FYI looks like this (I will just disable the test):
```
==> Test Info: seed=8E1FA201BA08A5B8; jvms=4; suites=641
Suite: org.elasticsearch.index.query.GeoPolygonQueryBuilderTests
  2> REPRODUCE WITH: gradle :core:test -Dtests.seed=8E1FA201BA08A5B8 -Dtests.class=org.elasticsearch.index.query.GeoPolygonQueryBuilderTests -Dtests.method="testToQuery" -Des.logger.level=WARN -Dtests.security.manager=true -Dtests.locale=be-BY -Dtests.timezone=America/Panama
ERROR   0.04s J0 | GeoPolygonQueryBuilderTests.testToQuery <<< FAILURES!
   > Throwable #1: java.lang.IllegalArgumentException: invalid minLon -180.000001
   > 	at __randomizedtesting.SeedInfo.seed([8E1FA201BA08A5B8:79E4A03FCB8B6052]:0)
   > 	at org.apache.lucene.util.GeoRect.<init>(GeoRect.java:29)
   > 	at org.apache.lucene.util.GeoUtils.polyToBBox(GeoUtils.java:221)
   > 	at org.apache.lucene.search.GeoPointInPolygonQuery.<init>(GeoPointInPolygonQuery.java:60)
   > 	at org.elasticsearch.index.query.GeoPolygonQueryBuilder.doToQuery(GeoPolygonQueryBuilder.java:152)
   > 	at org.elasticsearch.index.query.AbstractQueryBuilder.toQuery(AbstractQueryBuilder.java:78)
   > 	at org.elasticsearch.index.query.AbstractQueryTestCase.testToQuery(AbstractQueryTestCase.java:516)
   > 	at org.elasticsearch.index.query.GeoPolygonQueryBuilderTests.testToQuery(GeoPolygonQueryBuilderTests.java:115)
   > 	at java.lang.Thread.run(Thread.java:745)

```

**Comment by nknize** [comment]
Thank @rmuir  I pushed a fix to https://github.com/apache/lucene-solr/commit/62dfc815b030ca051379a12061fb0a9aa98ca09c

**Comment by clintongormley** [comment]
@nknize is this closed by https://github.com/elastic/elasticsearch/pull/16615 ?

**Comment by nknize** [comment]
Closed by #16615
